pilot/docker/envoy_pilot.yaml.tmpl

```diff
@@ -73,17 +73,17 @@ static_resources:
             name: envoy.file_access_log
           http_filters:
           - config:
-              default_destination_service: istio-pilot.{{ .PodNamespace }}.svc.cluster.local
+              default_destination_service: istio-pilot.{{ .PodNamespace }}.svc.{{  .ClusterDomainName }}
```

Comment:
you could also omit the cluster.local and stick with just istio-pilot.istio-system

Reply:
I am curious why it has not been done before, was there a particular reason to hard code cluster.local and now that reason is gone?

Reply:
No, downstream systems expect fully qualified names.